### PR TITLE
A4A > Migrations: Implement consolidated commissions view

### PR DESCRIPTION
**Pull request by yashwin**
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1444

## Proposed Changes

This PR implements a consolidated commission view. It currently uses hard-coded data, which will be replaced by the actual data later in a different PR.

## Why are these changes being made?

* To display the Migration commissions & Sites pending review.

## Testing Instructions

* Open the A4A live link > Go to Migrations: Commissions and verify that the below screen is displayed.

Screenshot 2024-11-11 at 1 09 22 PM

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+			case 2:
+				return 'Q2';
+			case 3:
+				return 'Q3';
+			case 4:
+				return 'Q4';
+			default:
+				return '';
+		}
+	};
```
Comment (vitozev) [inline client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx]: This function could be simplified to:

```
const getCurrentQuarter = () => {
    const month = new Date().getMonth();
    const quarter = Math.floor( month / 3 ) + 1;
    return `Q${quarter}`;
};
```

Comment (yashwin) [reply client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx]: Thanks for the review, @vitozev!

I have updated the logic: https://github.com/Automattic/wp-calypso/pull/96225/commits/dcc278064b76856fe71505839c86f12723526938

Could you please take another look?

File: client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import { useTranslate } from 'i18n-calypso';
+
+import './style.scss';
+
+const getCurrentQuarter = () => {
+	const currentMonth = new Date().getMonth();
+	const quarter = Math.ceil( ( currentMonth + 1 ) / 3 );
+	return `Q${ quarter }`;
+};
+export default function MigrationsConsolidatedCommissions() {
```
Comment (vitozev) [inline client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx:12]: Nitpick: Let's insert an empty after the function declaration.

**Review by vitozev — APPROVED** [review]
LGTM!

File: client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx (hunk 2 above)
Comment (yashwin) [reply client/a8c-for-agencies/sections/migrations/consolidated-commissions/index.tsx:12]: Fixed it: https://github.com/Automattic/wp-calypso/pull/96225/commits/fbed8c0b6f4dbb1ac7dfb25e60a5dc6748342779

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/17000886

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @yashwin for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/17000886).

**Pull request merged**